portal-impl/src/portal.properties

```diff
@@ -6778,6 +6783,22 @@
     #
     feature.flag.LPS-134060.type=beta
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER5__NUMBER4__NUMBER5__NUMBER1__NUMBER5__PERIOD_DESCRIPTION
+    #
+    feature.flag.LPS-154515.description=Documents manager will be able to set permissions from folders to folders by inheritance.
```

Comment:
I think the message should be this: 
Enable documents manager to propagate folders' permissions to their children folders.